test/functional/test_framework/wallet.py

```diff
@@ -65,15 +65,20 @@ def send_self_transfer(self, *, fee_rate=Decimal("0.003"), from_node, utxo_to_sp
         assert send_value > 0
 
         tx = CTransaction()
-        tx.vin = [CTxIn(COutPoint(int(utxo_to_spend['txid'], 16), utxo_to_spend['vout']))]
+        tx.vin = [CTxIn(COutPoint(int(utxo_to_spend['txid'], 16), utxo_to_spend['vout']), nSequence = seq_num)]
         tx.vout = [CTxOut(int(send_value * COIN), self._scriptPubKey)]
+        tx.nVersion = 2
         tx.wit.vtxinwit = [CTxInWitness()]
         tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_TRUE])]
         tx_hex = tx.serialize().hex()
 
         tx_info = from_node.testmempoolaccept([tx_hex])[0]
-        self._utxos.append({'txid': tx_info['txid'], 'vout': 0, 'value': send_value})
-        from_node.sendrawtransaction(tx_hex)
+        if send_tx:
+            if default_fee:
```

Comment:
This should be called `bypass_maxfeerate`. But really, since it's a parameter for `sendrawtransaction`, why isn't the caller doing a `send_tx=False` and then submitting it themselves?

Reply:
So this was following Darius's [comment](https://github.com/bitcoin/bitcoin/pull/21014#discussion_r575909995). I could only append tx in self.utxos after sending a transaction. If I try to simply send transaction from somewhere else, I wouldn't be able to append it in the internal wallet.py list